File: TOKILearning/v1/1B/toki1B4.pas
{
        toki1B2.pas
        
        Copyright 2012 Rakha Kanz Kautsar <rkkautsar@Portege-T110>
        
        This program is free software; you can redistribute it and/or modify
        it under the terms of the GNU General Public License as published by
        the Free Software Foundation; either version 2 of the License, or
        (at your option) any later version.
        
        This program is distributed in the hope that it will be useful,
        but WITHOUT ANY WARRANTY; without even the implied warranty of
        MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
        GNU General Public License for more details.
        
        You should have received a copy of the GNU General Public License
        along with this program; if not, write to the Free Software
        Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
        MA 02110-1301, USA.
        
        
}


program untitled;

uses crt;
var a:1..100000;

BEGIN
	readln(a);
	case a of
		1..9: writeln('satuan');
		10..99: writeln('puluhan');
		100..999: writeln('ratusan');
		1000..9999: writeln('ribuan');
		10000..99999: writeln('puluhribuan');
	else writeln('ratusribuan');
	end;
END.

File: TOKILearning/v1/1B/toki1b4.cpp
#include <iostream>
using namespace std;

int main()
{
	int a;
	cin>>a;
	if(a/10==0) cout<<"satuan\n";
	else if(a/100==0) cout<<"puluhan\n";
	else if(a/1000==0) cout<<"ratusan\n";
	else if(a/10000==0) cout<<"ribuan\n";
	else if(a/100000==0) cout<<"puluhribuan\n";
	else cout<<"ratusribuan\n";
	return 0;
}

